clowdapp.yaml

```diff
@@ -69,6 +69,12 @@ objects:
         # Should be a secret ?
         - name: SECRET_KEY_BASE
           value: "secret_key_base"
+        - name: SETTINGS__SLACK_WEBHOOK
+          valueFrom:
+            secretKeyRef:
+              name: compliance-backend
```

Comment:
Where is the secret defined?

Reply:
Vault, it's being pulled via app-interface.